clang_tidy/clang_tidy.bzl

```diff
@@ -91,9 +91,13 @@ def _run_tidy(
         for i in compilation_context.includes.to_list():
             args.add("-I" + i)
 
-        args.add_all(compilation_context.quote_includes.to_list(), before_each = "-iquote")
-
-        args.add_all(compilation_context.system_includes.to_list(), before_each = "-isystem")
+        for i in compilation_context.quote_includes.to_list() + compilation_context.system_includes.to_list():
+            if i.startswith(("external", "/", "src/third_party/")):
+                args.add("-isystem")
```

Comment:
This raises a question, does code we generate for third party count as our code our third party code (angle brackets are in question here)

Reply:
For example we generate some wiredtiger headers to our bazel-out dir

Reply:
I switched to an inclusive condition which mitigates this issue